src/SurveyService/Startup.cs

```diff
@@ -0,0 +1,206 @@
+
+    private void ConfigureMassTransit(IServiceCollection services)
+    {
+      (string username, string password) = RabbitMqCredentialsHelper
+        .Get(_rabbitMqConfig, _serviceInfoConfig);
+
+      services.AddMassTransit(busConfigurator =>
+      {
+        busConfigurator.UsingRabbitMq((context, cfg) =>
+          {
+            cfg.Host(_rabbitMqConfig.Host, "/", host =>
+              {
+                host.Username(username);
+                host.Password(password);
+              });
+
+            ConfigureEndpoints(context, cfg, _rabbitMqConfig);
+          });
+
+        ConfigureConsumers(busConfigurator);
+
+        busConfigurator.AddRequestClients(_rabbitMqConfig);
+      });
+
+      services.AddMassTransitHostedService();
+    }
+
+    private void ConfigureConsumers(IServiceCollectionBusConfigurator x)
+    {
+    }
+
+    private void ConfigureEndpoints(
+        IBusRegistrationContext context,
+        IRabbitMqBusFactoryConfigurator cfg,
+        RabbitMqConfig rabbitMqConfig)
+    {
+    }
+
+    #endregion
+
+    #endregion
+
+    public Startup(IConfiguration configuration)
+    {
+      Configuration = configuration;
+
+      _serviceInfoConfig = Configuration
+        .GetSection(BaseServiceInfoConfig.SectionName)
+        .Get<BaseServiceInfoConfig>();
+
+      _rabbitMqConfig = Configuration
+        .GetSection(BaseRabbitMqConfig.SectionName)
+        .Get<RabbitMqConfig>();
+
+      Version = "1.0.0.0";
+      Description = "SurveyService is an API that intended to work with surveies.";
+      StartTime = DateTime.UtcNow;
+      ApiName = $"LT Digital Office - {_serviceInfoConfig.Name}";
+    }
+
+
+    public void ConfigureServices(IServiceCollection services)
+    {
+      services.AddCors(options =>
+      {
+        options.AddPolicy(
+          CorsPolicyName,
+          builder =>
+          {
+            builder
+              .AllowAnyOrigin()
+              .AllowAnyHeader()
+              .AllowAnyMethod();
+          });
+      });
+
+      services.AddHttpContextAccessor();
+
+      string dbConnStr = ConnectionStringHandler.Get(Configuration);
+
+      services.AddDbContext<SurveyServiceDbContext>(options =>
+      {
+        options.UseSqlServer(dbConnStr);
+      });
+
+      services.AddHealthChecks()
+        .AddRabbitMqCheck()
+        .AddSqlServer(dbConnStr);
+
+      services.Configure<TokenConfiguration>(Configuration.GetSection("CheckTokenMiddleware"));
+      services.Configure<BaseServiceInfoConfig>(Configuration.GetSection(BaseServiceInfoConfig.SectionName));
+      services.Configure<BaseRabbitMqConfig>(Configuration.GetSection(BaseRabbitMqConfig.SectionName));
+
+      services.AddBusinessObjects();
+
+      ConfigureMassTransit(services);
+
+      services
+        .AddControllers()
+        .AddJsonOptions(options =>
+        {
+          options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
+        })
+        .AddNewtonsoftJson();
+    }
+
+
```

Comment:
remove empty line: for separating methods use one empty line (not two)

Reply:
done